pom.xml

```diff
@@ -744,6 +743,11 @@ under the License.
 			<build>
 				<pluginManagement>
 					<plugins>
+						<plugin>
+							<groupId>org.codehaus.mojo</groupId>
+							<artifactId>build-helper-maven-plugin</artifactId>
+							<version>1.7</version>
```

Comment:
Why are do we pin it to 1.7, and not 1.12 (the version that was previously used when the Scala 2.12 profile is enabled)?

Reply:
because quite a few modules, primarily scala reliant ones, were overriding the version to 1.7 already, and it apparently did not cause issues.